### PR TITLE
add manifest.json file

### PR DESCRIPTION
**Pull request by scottinet**
# Description

Add a `manifest.json` file specifying the supported Kuzzle versions

# Boyscout

Dependencies update

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -52,11 +49,8 @@ services:
     image: redis:3.2
 
   kdc-elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:5.4.1
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.6.10
```
Comment (Aschen) [inline docker-compose.yml]: We aren't using `kuzzleio/elasticsearch` anymore ?

Comment (scottinet) [reply docker-compose.yml]: I believe our es image is deprecated

Comment (Aschen) [reply docker-compose.yml]: I'm not quite sure, @benoitvidis just updated the image to use the last elasticsearch version 10 days ago https://github.com/kuzzleio/kuzzle-containers/tree/master/elasticsearch

Comment (scottinet) [reply docker-compose.yml]: ah, you seem to be right
I'll fix that, then

**Pull request merged**